### ISSUE TITLE
Everytime i open the app, i need to register the wallet again

### ISSUE THREAD

**Issue by ufkhf** [flathub/org.getmonero.Monero#13]

**Comment by BigmenPixel0** [comment]
Do you store your wallet file in ~/Monero?

**Comment by ufkhf** [comment]
Yes i added a file system permission so the app can access a SSD to store the blockchain.


![image](https://user-images.githubusercontent.com/84050866/153718658-4b59fd55-d328-41fc-9821-1be42a1c3112.png)

**Comment by BigmenPixel0** [comment]
> Yes i added a file system permission so the app can access a SSD to store the blockchain.

Can you run application like this:
```
flatpak run -vv org.getmonero.Monero
```
and send the log here?

**Comment by ufkhf** [comment]
file:///home/k/Persoonlijke%20bestanden/Tijdelijk/Log.txt

**Comment by ufkhf** [comment]
[Log.txt](https://github.com/flathub/org.getmonero.Monero/files/8053972/Log.txt)

**Comment by BigmenPixel0** [comment]
> Log.txt

Do you have something in `~/.local/share/flatpak/overrides/global`?

**Comment by ufkhf** [comment]
Only this k@k:~/.local/share/flatpak/overrides$ ls
org.getmonero.Monero

**Comment by BigmenPixel0** [comment]
> Only this k@k:~/.local/share/flatpak/overrides$ ls
> org.getmonero.Monero

Can you try restoring the permissions for `org.getmonero.Monero`?

**Comment by BigmenPixel0** [comment]
Btw, does the ~/Monero directory exist?

**Comment by ufkhf** [comment]
If i try to restore permissions i have to register again.

And now after i reset the permissions the org.getmonero.Monero dissapeared.
![image](https://user-images.githubusercontent.com/84050866/153720007-72529ccd-2b26-4400-af43-0cba46200c33.png)

**Comment by BigmenPixel0** [comment]
> If i try to restore permissions i dont have to register again.
> 
> And now after i reset the permissions the org.getmonero.Monero dissapeared. ![image](https://user-images.githubusercontent.com/84050866/153720007-72529ccd-2b26-4400-af43-0cba46200c33.png)

So, no problems now? And if you add your permission, will you need to create a wallet again?
